src/main/java/mate/academy/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -46,6 +46,7 @@ public ErrorResponse handlePropertyReferenceException(
     }
 
     @ExceptionHandler({
+            RegistrationException.class,
             DataIntegrityViolationException.class,
             EntityAlreadyExistsException.class
     })
```

Comment:
I can partially agree with the `CONFLICT` status code here, but I'd personally use simple bad request for those exceptions, up to you